CMake/mlpack.cmake

```
@@ -528,11 +528,15 @@ endmacro()
 macro(find_mlpack_internal)
 
   set(CURRENT_PATH ${ARGN})
+
   if (CURRENT_PATH)
-    find_path(MLPACK_INCLUDE_DIR
-      NAMES mlpack.hpp
-      PATHS "${CURRENT_PATH}/deps/mlpack/src/include" "${CMAKE_CURRENT_SOURCE_DIR}/src/"
-      NO_DEFAULT_PATH)
+    file(GLOB MLPACK_SEARCH_PATHS
+      ${CMAKE_BINARY_DIR}/deps/mlpack-[0-9]*.[0-9]*.[0-9]*/src/)
```

```suggestion
      ${CURRENT_PATH}/deps/mlpack-[0-9]*.[0-9]*.[0-9]*/src/)
```
Should this be `CURRENT_PATH`?